test/unit_test/passes/pytorch/test_quantization_aware_training.py

```diff
@@ -2,18 +2,26 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
-from test.unit_test.utils import create_dataloader, get_pytorch_model
+from test.unit_test.utils import get_pytorch_model
 
+from torch.utils.data import DataLoader
+
+from olive.data.component.dataset import DummyDataset
 from olive.hardware.accelerator import AcceleratorSpec
 from olive.passes.olive_pass import FullPassConfig, create_pass_from_dict
 from olive.passes.pytorch.quantization_aware_training import QuantizationAwareTraining
 
 
+# TODO(shaahji): Remove this once QuantizationAwareTraining pass supports DataConfig
+def _create_dummy_dataloader(data_dir, batch_size=1, size=1, **kwargs):
```

Comment:
same here

Reply:
This was done in response to your comment about time it takes for training. Required to force a size of 1 for dataset.